modules/README.md

```diff
@@ -33,9 +33,12 @@ Modules that are still in development and less stable are labeled with the
 * **[vm-instance]** ![core-badge] : Creates one or more VM instances.
 * **[SchedMD-slurm-on-gcp-partition]** ![community-badge] : Creates a partition
   to be used by a [slurm-controller][schedmd-slurm-on-gcp-controller].
+* **[schedmd-slurm-gcp-v5-partition]** ![community-badge] ![experimental-badge] :
+  Creates a partition to be used by a [slurm-controller][schedmd-slurm-gcp-v5-controller].
 
 [vm-instance]: compute/vm-instance/README.md
 [schedmd-slurm-on-gcp-partition]: ../community/modules/compute/SchedMD-slurm-on-gcp-partition/README.md
+[schedmd-slurm-on-gcp-partition-v5]: ../community/modules/compute/schedmd-slurm-gcp-v5-partition/README.md
```

Comment:
How do we feel about the difference in capitalization between v4 and v5? I don't have any strong opinion, I just wanted to make sure the choice was conscious.

Reply:
This was discussed post standup a couple weeks back, though I think you might not have been there. This was the consensus, but there is room to update this if you have objections. 

I also noticed that this link is not correct, I'll update.

Reply:
Link has been updated.